requirements.txt

```diff
@@ -1,5 +1,5 @@
-apprise==1.8.1
+apprise==1.9.0
 ntplib==0.4.0
 pytz==2024.1  # Remove when this script only supports Python 3.9+
 requests==2.31.0
-seleniumbase==4.29.9
+seleniumbase>=4.30.3
```

Comment:
I would prefer to lock SB at a specific version rather than `>=` so there aren't any unexpected changes or different environments between users.